contracts/test/integration/LoanRepayment.t.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Vm} from "./../unit/utils/Vm.sol";
+import {DSTest} from "./../unit/utils/DSTest.sol";
+import {getCore, getAddresses} from "./../unit/utils/Fixtures.sol";
+import {IVolt, Volt} from "../../volt/Volt.sol";
+import {OtcEscrow} from "./../../utils/OtcEscrow.sol";
+import {MainnetAddresses} from "./fixtures/MainnetAddresses.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {IFeiPCVGuardian} from "../../pcv/IFeiPCVGuardian.sol";
+
+contract IntegrationTestLoanRepayment is DSTest {
+    IVolt private volt = IVolt(MainnetAddresses.VOLT);
+    IVolt private fei = IVolt(MainnetAddresses.FEI);
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    /// @notice amount of fei to pay off loan
+    uint256 public constant feiAmount = 10_170_000e18;
+
+    /// @notice amount of volt to receive upon repayment
+    uint256 public constant voltAmount = 10_000_000e18;
+
+    /// @notice escrow contract
+    OtcEscrow public otcEscrow = OtcEscrow(MainnetAddresses.OTC_LOAN_REPAYMENT);
+
+    address public feiTcTimelock = MainnetAddresses.FEI_TC_TIMELOCK;
+    address public voltTimelock = MainnetAddresses.VOLT_TIMELOCK;
```

Comment:
Should this be updated from the deprecated VOLT_TIMELOCK to the new `TIMELOCK_CONTROLLER` ?